packages/logger/examples/usage/index.ts

```diff
@@ -23,22 +23,19 @@ const fluentLogger1 = createLogger({
     }),
   ],
 });
-function encrypt(text: string, password: string): string {
-  const cipher = crypto.createCipher('aes-256-ccm', password);
-  let crypted = cipher.update(text, 'utf8', 'hex');
-  crypted += cipher.final('hex');
-  return crypted;
-}
 // demonstrates an encrypted logger
 const fluentLogger2 = createLogger({
   formatters: [
     (info) => {
       return {
         ...info,
-        message: `enc.${encrypt(info.message, 'supersecretstuff')}`,
+        message: `${info.message.substr(0, 4)}${'*'.repeat(
```

Comment:
doesn't exactly show an encrypted logger- i think this could be generalised as custom message formatters

Reply:
I dont understand ? Here the idea is to demonstrate a way to encrypt something ? Basically we dont care about which lib we use to encrypt but we care about how we need to set up the logger to encrypt data ?

Reply:
yeah, it was supposed to demonstrate a way to inject a custom formatter, but l26 might be misleading, this is more like redacted than encrypted

Reply:
ok I added a comment (I replaced the previous because the demo wasn't working)